binaryaudit/RPM.py

```diff
@@ -0,0 +1,27 @@
+# Group packages based on source RPM
+
+import json
+import subprocess
+
+def groupPackages(dirPath):        
+    ''' Groups packages based on source RPM and outputs to JSON file.
+    
+    Parameters:
+        dirPath (str): The path to the input directory.  
+    '''
+    file = open("RPMList.txt", "r")
```

Comment:
This intermediate file is not necessary. It should be fine to use something like below right at the place:

```python
for filename in os.listdir(directory):
    f = os.path.join(directory, filename)
    if os.path.isfile(f):
        print(f)
```

This will also render the file `run.py` obsolete. In general, if there's a possibility to use a specific language API, it should be preferred over doing system call to a shell command.

Thanks